ci/daily_tell_slack.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+parameters:
+  success-message: '"$(Agent.JobName) passed: $COMMIT_LINK"'
+
+steps:
+- bash: |
+    set -euo pipefail
+    eval "$(dev-env/bin/dade assist)"
+    COMMIT_TITLE=$(git log --pretty=format:%s -n1)
+    COMMIT_LINK="<https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|$COMMIT_TITLE>"
+    if [ "$(Agent.JobStatus)" != "Succeeded" ]; then
+        MESSAGE=":fire: :fire: <!here> :fire: :fire:\n$(Agent.JobName) *FAILED*: $COMMIT_LINK\n:fire: :fire:"
+    else
+        MESSAGE=${{ parameters['success-message'] }}
+    fi
+    if [ "$(variables['Build.SourceBranchName'])" = "master" ]; then
+        curl -XPOST \
+             -i \
+             -H 'Content-type: application/json' \
+             --data "{\"text\":\"$MESSAGE\n\"}" \
+             $(Slack.team-daml)
+    else
+        echo "{\"text\":\"$MESSAGE\n\"}"
```

Comment:
Doesn’t matter much but you could factor this out to ensure that you are actually printing the same thing that you would send to slack.

Reply:
Good point, will do.